identify/extensions.py

```diff
@@ -246,18 +253,24 @@
     'BUILD': {'text', 'bazel'},
     'BUILD.bazel': {'text', 'bazel'},
     'CMakeLists.txt': EXTENSIONS['cmake'],
+    'CHANGELOG': EXTENSIONS['txt'],
+    'CODE_OF_CONDUCT': EXTENSIONS['txt'],
+    'CONTRIBUTING': EXTENSIONS['txt'],
     'COPYING': EXTENSIONS['txt'],
     'Dockerfile': {'text', 'dockerfile'},
     'Gemfile': EXTENSIONS['rb'],
+    'GPL': EXTENSIONS['txt'],
```

Comment:
is this actually used? I've never seen this

Reply:
Oops, I saw this in a project and misremembered it as the recommended practice for adding the LGPL, when the latter is in fact conventionally added as `COPYING.LESSER` per the FSF [GPL Howto](https://www.gnu.org/licenses/gpl-howto.html), so I substituted that instead.